app/uk/gov/hmrc/helptosavefrontend/config/frontendGlobal.scala

```diff
@@ -16,12 +16,13 @@
 
 package uk.gov.hmrc.helptosavefrontend
```

Comment:
ide  warning : package name doesnt correspond to directory structure, ...`config`